cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function uses the `namespace` argument, but ensure that the included `cinema.urls` and `user.urls` modules define an `app_name` attribute. This is necessary for namespacing to work correctly.